order/src/main/java/com/yas/order/controller/CartItemController.java

```diff
@@ -14,7 +19,14 @@ public class CartItemController {
     private final CartItemService cartItemService;
 
     @PostMapping("/storefront/cart/items")
-    public void addCartItem(@Valid @RequestBody CartItemPostVm cartItemPostVm) {
+    public ResponseEntity<Void> addCartItem(@Valid @RequestBody CartItemPostVm cartItemPostVm) {
         cartItemService.addCartItem(cartItemPostVm);
+        return ResponseEntity.ok().build();
+    }
+
+    @PutMapping("/storefront/cart/items/{productId}")
```

Comment:
Since we're only updating the `quantity` field (partial update), consider using a PATCH request instead.

Reply:
Thanks for the suggestion. The design document requires this API to use `PUT` for updates. You can refer to it for more information. https://github.com/nashtech-garage/yas/issues/1099